app/uk/gov/hmrc/eoricommoncomponent/frontend/views/match_nino_row_individual.scala.html

```diff
@@ -34,15 +37,15 @@
         Map.empty,
         Some(label),
         None,
-        '_divClass -> "govuk-form-group inline",
-        '_legend -> messages("cds.matching.nino.row.header"),
-        '_isLegendH1 -> true
+        Symbol("_divClass") -> "govuk-form-group inline",
+        Symbol("_legend") -> messages("cds.matching.nino.row.header"),
+        Symbol("_isLegendH1") -> true
     )
 }
 
 @label = {
-<p class="govuk-body">@messages("cds.matching.nino.row.description")</p>
-<p class="govuk-body">@Html(messages("cds.matching.nino.nino.hint"))</p>
+    <p class="govuk-body">@messages("cds.matching.nino.row.description")</p>
```

Comment:
Since we're changing this anyway, could we improve it using the paragraph helper, it'll look cleaner